runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -130,8 +130,8 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
     private <R> R accessInner(ICorfuSMRAccess<R, T> accessMethod,
                               Object[] conflictObject, boolean isMetricsEnabled) {
         if (TransactionalContext.isInTransaction()) {
-            log.trace("Access[{}] conflictObj={} @TX[{}]", this, conflictObject,
-                    Utils.toReadableID(TransactionalContext.getCurrentContext().getTransactionID()) );
+            log.debug("Object Access[{}] conflictObj={} @TX[{}]", this, conflictObject,
```

Comment:
is Object really necessary here? It just makes the log message unnecessarily long...